common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -240,8 +256,12 @@ def create_event(self, knowledge_node_uid, student_uid, event_type, **data):
         payload.update(data)
 
         # Send request
-        response = requests.post(url, headers=self.request_headers, data=payload)
-        event = json.loads(response.content)
+        try:
+            event = {}
+            response = requests.post(url, headers=self.request_headers, data=payload)
+            event = json.loads(response.content)
+        except requests.ConnectionError:
+            log.warning("Could not create event of type {event_type} for unit identified by knowledge_node_uid {knowledge_node_uid} and student identified by student_uid {student_uid}.".format(event_type=event_type, student_uid=student_uid, knowledge_node_uid=knowledge_node_uid))
```

Comment:
@macornwell This line is too long; please add some line breaks to make it shorter.